README.md

```diff
@@ -30,34 +36,32 @@ Needle requires the following permissions to function, along with the `applicati
 -   [x] Create public threads
 -   [x] Read message history
 
-You can use this link to invite Needle, replacing `<APP ID>` with your bot's application ID.
-
-```
-https://discord.com/oauth2/authorize?client_id=<APP ID>&permissions=309237713920&scope=bot%20applications.commands
-```
+You can use this link to invite your self-hosted version of Needle, replacing `<APP ID>` with your bot's application ID: `https://discord.com/oauth2/authorize?client_id=<APP ID>&permissions=309237713920&scope=bot%20applications.commands`
 
 ### 🐳 Docker
 
-Needle has an [official Docker image](https://github.com/MarcusOtter/discord-needle/pkgs/container/discord-needle). Releases are tagged by their minor & patch version (eg. `2.0.0` & `2.0`), with the latest release tagged `latest`. Branches are tagged by their name.
+Needle has an [official Docker image](https://github.com/MarcusOtter/discord-needle/pkgs/container/discord-needle). Releases are tagged by their minor & patch version (e.g. `2.0.0` & `2.0`), with the latest release tagged `latest`. Branches are tagged by their name. To run the image, write the following command, replacing `token` with your bot's token:
 
 ```sh
 docker run -d --name Needle --env=DISCORD_API_TOKEN=token discord-needle ghcr.io/MarcusOtter/discord-needle:latest
```

Comment:
Mobile makes this hard but do you need a space after env=?

Reply:
I don't think there should be a space after `env=`, Nick might know more

Reply:
Actually according to the docs there shouldn't be an equal sign at all: https://docs.docker.com/engine/reference/commandline/run/
![image](https://user-images.githubusercontent.com/35617441/168488252-bacba05f-28b5-4fc0-81a5-ffc372963fe1.png)